CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Version 1.10.0
 
+* **[Improvement]** Specified minimum cocoapods version in podspec to 1.10.0.
```

Comment:
This should be under ## Version 1.10.1 (Under development). Version 1.10.0 is already released.

Reply:
done